src/main/java/net/torocraft/signedit/SignEdit.java

```diff
@@ -26,32 +26,40 @@
   public static final String MODID = "signedit";
   public static final String VERSION = "1.12.2-4";
   public static final String MODNAME = "SignEdit";
+  
+  public static Configuration cfg;
+  public static Item editor;
 
   @EventHandler
   public void preInit(FMLPreInitializationEvent e) {
     MinecraftForge.EVENT_BUS.register(this);
+    
+    cfg = new Configuration(new File("config/signedit.cfg"));
+    cfg.load();
+    
+    String cfgEditor = cfg.get("SignEdit", "editor", "minecraft:sign", "The player must hold this item to edit signs. Enter in the format modid:itemname. Default: 'minecraft:sign'. Use '*' to always allow editing regardless of held items.").getString();
+    
+    if (cfgEditor.equals("*")) editor = null;
+    else {
+      editor = Item.REGISTRY.getObject(new ResourceLocation(cfgEditor));
+      if (editor == null) editor = Items.SIGN;
+    }
+    
+    cfg.save();
   }
 
   @SubscribeEvent
   public void editSign(RightClickBlock event) {
-    if (event.getEntityPlayer().isSneaking()) {
-      return;
-    }
-
-    if (!isHoldingSign(event.getEntityPlayer())) {
-      return;
-    }
-
+    if (event.getEntityPlayer().isSneaking() || !isHoldingEditor(event.getEntityPlayer())) return;
```

Comment:
I don't think any thing changed here to add this feature, so this should be restored